guide/ops/locations/customizing_location_ports.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Customizing Location Ports
+layout: website-normal
+---
+
+## Opening ports during startup.
+
+Use the `org.apache.brooklyn.location.jclouds.networking.SharedLocationSecurityGroupCustomizer` location customizer.
+Example:
+
+    name: ports @ AWS
+    location:
+      jclouds:aws-ec2:us-west-2:
+        customizers:
+        - $brooklyn:object:
+            type: org.apache.brooklyn.location.jclouds.networking.SharedLocationSecurityGroupCustomizer
+            object.fields: {tcpPortRanges: ["900-910", "915", "22"], udpPortRanges: ["100","200-300"], cidr: "82.40.153.101/24"}
+        - $brooklyn:object:
+            type: org.apache.brooklyn.location.jclouds.networking.SharedLocationSecurityGroupCustomizer
+            object.fields: {tcpPortRanges: ["900-910", "915", "22"], udpPortRanges: ["100","200-300"], cidr: "10.40.153.101/16"}
```

Comment:
Perhaps just show the one example `SharedLocationSecurityGroupCustomizer`. The duplication (to open it to the second cird, with all the same ports) looks cumbersome and is probably not the most common pattern.